cmd/sebak/cmd/run.go

```diff
@@ -41,8 +41,8 @@ var (
 		fmt.Sprintf("%s://%s:%d", defaultNetwork, defaultHost, defaultPort),
 	)
 	flagStorageConfigString string
-	flagTLSCertFile         string = sebakcommon.GetENVValue("SEBAK_TLS_CERT", "sebak.crt")
-	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "sebak.key")
+	flagTLSCertFile         string = sebakcommon.GetENVValue("SEBAK_TLS_CERT", "")
+	flagTLSKeyFile          string = sebakcommon.GetENVValue("SEBAK_TLS_KEY", "")
```

Comment:
Why ? It's useful to have

Reply:
I think those should be two separate PR.
In addition, the logic here seems to be that TLS is used based on the content of those variables.
Since it reads from environment variable, that is problematic.
The choice to use TLS should be based solely on the scheme of the endpoint.
So we can use `http` even if a user provide a value in those variables.

Reply:
@Geod24 Yes, right. HTTP and HTTPS is determined by endpoint scheme.

Reply:
So can you revert this ?

Reply:
@Geod24 In this PR, endpoint determine the HTTP and HTTPS.

Reply:
Exactly. So you don't need to change `flagTLSKeyFile` because it will only be used when endpoint set HTTPS.

Reply:
@Geod24 I bothered you. :) As you pointed, the code will be reverted. Thanks.